modules/transaction.rb

```diff
@@ -0,0 +1,23 @@
+module Transaction
+  def deposit_money(money)
+      if money > 0
+        @balance += money 
+        puts "Deposited $ #{money}. Your account balance is $ #{@balance}."
+      else  
+        puts "You can't deposit a negative amount!"
+      end 
```

Comment:
Here we can also do the negative check first and return early from the method:
```suggestion
      if money <= 0
        puts "You can't deposit a negative amount!"
        return
      end
```